source/codegen/service_helpers.py

```diff
@@ -426,6 +426,11 @@ def get_driver_service_readiness(config: dict) -> str:
     return to_cpp_readiness(readiness)
 
 
+def is_restricted_driver_service(config):
+    """Whether the driver service is restricted."""
+    return config.get("restricted_driver_service", False)
```

Comment:
I think @madrasm had a different representation of this concept? Using `code-readiness`?

Reply:
Yeah, can you change the config.py to use `"code_readiness": "RestrictedNextRelease"` like what's done in https://github.com/ni/grpc-device/pull/689. You should also be able to re-use that version of `is_restricted_driver_service(config)` instead.

Reply:
Looks like this got fixed in the merge.